frontends/main/src/app-pages/ChannelPage/UnitChannelTemplate.tsx

```diff
@@ -126,7 +135,20 @@ const UnitChannelTemplate: React.FC<UnitChannelTemplateProps> = ({
               <ChannelHeader>
                 <VisuallyHidden>{channel.data?.title}</VisuallyHidden>
                 {channel.data ? (
-                  <UnitLogo alt="" src={displayConfiguration.logo} />
+                  <>
+                    <DesktopOnly>
+                      <PlatformLogoInverted
+                        unitCode={name as OfferedByEnum}
+                        height={50}
+                      />
+                    </DesktopOnly>
+                    <MobileOnly>
+                      <PlatformLogoInverted
+                        unitCode={name as OfferedByEnum}
+                        height={40}
+                      />
+                    </MobileOnly>
```

Comment:
Here we produce for each side of the width breakpoint and use the css media query to hide one. This is wordy, though needed for varying the height with only CSS. Where the images are in the repo we can generally pull them in as static imports, so don't need to specify the dimensions, though in this case the ol-components PlatformLogo is not allowed to import from outside its workspace. Further, the width and height on <Image> components do not size the image itself - it is just to block space to prevent layout shift, which can be done with CSS. Here though the logos are of different sizes so we need to hardcode the dimensions (here providing fixed height with width calculated in PlatformLogo).